pkg/plugin/odata/functions.go

```diff
@@ -1,8 +1,13 @@
 package odata
 
-import "fmt"
+import (
+	"fmt"
+	"time"
+)
 
+// Maps OData property types to Grafana Field type
 func ToArray(propertyType string) interface{} {
+	// TODO: Add field labels here?
```

Comment:
TODO?

Reply:
ok, got the point. Would it be better to define an empty/zero property instead?

Reply:
See new comment for LabelsFromString in datasource.go

Reply:
Nevertheless, the question of the TODO arises at this point. This is only about mapping a property type to a suitable Go array for the data frames. Remove TODO? Create an issue so that we can take a look at the topic of labeling?

Reply:
You are right, even if we start labeling more properties this would not be the right place. 
I will remove the TODO and create an issue regarding labeling.

Reply:
Created issue #69